backend/src/nodes/image_nodes.py

```diff
@@ -155,22 +155,32 @@ def run(
         img = (np.clip(img, 0, 1) * 255).round().astype("uint8")
 
         os.makedirs(base_directory, exist_ok=True)
-
-        # Write image with opencv if path is ascii, since imwrite doesn't support unicode
-        # This saves us from having to keep the image buffer in memory, if possible
-        if full_path.isascii():
-            status = cv2.imwrite(full_path, img)
+        # Any image not supported by cv2, will be handled by pillow.
+        if extension not in ["png", "jpg", "gif", "tiff", "webp"]:
+            status = 1  # spoof
```

Comment:
@joeyballentine Looking at this, why do we return anything at all? This node doesn't have any outputs, and no other code in the backend is using this node otherwise (I looked). Also, instead of returning a status code that nobody checks, the OpenCV code should probably just throw if an image couldn't be saved.

@emarron I'm sorry for spamming your PR. This is not something you have to change. It's a problem with our code.

Reply:
I did that when I first made this node thinking at some point we might wanna have branching on if it failed or not, and I've just never removed this

Reply:
*If* we want to allow failable nodes in the future, we should probably do it in a way that all nodes can use it.